packages/components/src/local/mapping/index.tsx

```diff
@@ -73,6 +73,7 @@ export const Mapping: React.FC<PropTypes> = ({
   /* Initialise states */
   const [showMapBar, setShowMapBar] = useState(false)
   const [currentForm, setCurrentForm] = useState('')
+  const [zoomLevel, setZoomLevel] = useState(zoom || 0)
```

Comment:
A zoom-level hook could be useful in other features being able to decide whether to show themselves or not.